homeassistant/components/netatmo/climate.py

```diff
@@ -169,42 +131,35 @@ class NetatmoThermostat(NetatmoBase, ClimateEntity):
     _attr_hvac_mode = HVACMode.AUTO
     _attr_max_temp = DEFAULT_MAX_TEMP
     _attr_preset_modes = SUPPORT_PRESET
-    _attr_supported_features = (
-        ClimateEntityFeature.TARGET_TEMPERATURE | ClimateEntityFeature.PRESET_MODE
-    )
+    _attr_supported_features = SUPPORT_FLAGS
     _attr_target_temperature_step = PRECISION_HALVES
     _attr_temperature_unit = TEMP_CELSIUS
 
-    def __init__(
-        self, data_handler: NetatmoDataHandler, room: pyatmo.climate.NetatmoRoom
-    ) -> None:
+    def __init__(self, netatmo_device: NetatmoRoom) -> None:
         """Initialize the sensor."""
         ClimateEntity.__init__(self)
-        super().__init__(data_handler)
+        super().__init__(netatmo_device.data_handler)
 
-        self._room = room
+        self._room = netatmo_device.room
         self._id = self._room.entity_id
+        self._home_id = self._room.home.entity_id
 
         self._signal_name = f"{CLIMATE_STATE_CLASS_NAME}-{self._room.home.entity_id}"
```

Comment:
Every PR needs to be working on its own when merged even if merging into a separate target branch. The climate platform seems to have issues.

Reply:
Linting failed in the last CI run before merging.

Reply:
Ah, right. I fixed those in the new branch. Those appeared after rebasing. Not exactly sure what I screwed up there. The new branch is green and works/behaves as expected.

Non the less, I totally get your point. If you want me to revert merges and fix things in the original branches, let me know.

Reply:
No, don't revert. Just a comment for the future. Thanks!